README.md

```diff
@@ -17,42 +17,76 @@ The [complete documentation](https://python-poetry.org/docs/) is available on th
 ## Installation
 
 Poetry provides a custom installer that will install `poetry` isolated
-from the rest of your system by vendorizing its dependencies. This is the
-recommended way of installing `poetry`.
+from the rest of your system.
```

Comment:
Not a blocker, but should we link to our documentation here instead of maintaining duplication information here? Might be good for the brevity of the README as well.